hardhat.config.ts

```diff
@@ -35,7 +35,7 @@ interface EnvOptions {
 const {
     TENDERLY_TESTNET_PROVIDER_URL = '',
     VERIFY_API_KEY = '',
-    GAS_PRICE: gasPrice = 'auto',
+    GAS_PRICE = 'auto',
```

Comment:
Unrelated to this PR, but you can also consider using v2 transactions by setting max base and priority fees instead (== it'll cheapen the deployments, since currently both are effectively set to the same value)

Reply:
Why did you remove the alias of `gasPrice`? Without it, you indeed have to duplicate `gasPrice: GAS_PRICE` multiple times below...

Reply:
@ivanzhelyazkov that's to you

Reply:
fixed